source/Calamari/Kubernetes/Authentication/AwsCliAuth.cs

```diff
@@ -129,6 +149,34 @@ void SetKubeConfigAuthenticationToAwsCli(string user, string clusterName, string
             log.AddValueToRedact(token, "<token>");
             kubectl.ExecuteCommandAndAssertSuccess(arguments.ToArray());
         }
+        
+        void SetKubeConfigAuthenticationToAwsCliUsingExec(string user, string clusterName, string region)
+        {
+            var oidcJwt = deploymentVariables.Get(AccountVariables.Jwt);
+            
+            var apiVersion = GetKubeCtlAuthApiVersion();
```

Comment:
We changed to use the kubectl approach for getting the correct api version. This should really only be `v1beta1` as `v1alpha1` was deprecated in `1.24` of Kubernetes (which is not supported on AWS)